### PR TITLE
v1.17.0

### PR DESCRIPTION
**Pull request by ashie**
**Which issue(s) this PR fixes**: 
Fixes #

**What this PR does / why we need it**: 

**Docs Changes**:

**Release Note**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+* Minor code refactoring
+  https://github.com/fluent/fluentd/pull/4294
+  https://github.com/fluent/fluentd/pull/4299
+  https://github.com/fluent/fluentd/pull/4302
+  https://github.com/fluent/fluentd/pull/4320
+* Raise minimum required ruby version
+  https://github.com/fluent/fluentd/pull/4288
+
+### Bug Fix
+
```
Comment (daipom) [inline CHANGELOG.md:44]:
```suggestion
```

Comment (ashie) [reply CHANGELOG.md]: Oops, thanks pointing out it!

**Review by daipom — APPROVED** [review]
LGTM.
Thanks!

**Comment by ashie** [comment]
I'll merge this after re-checking the list.

**Pull request merged**